### DISCUSSION TITLE
[AssetManager] how to require Hyperscript?

### DISCUSSION THREAD

**Discussion by back-2-95** [symfony/symfony#52562] [Q&A]
I'd like to require https://hyperscript.org/docs/#install with Asset Mapper.

Requiring works nicely with `bin/console importmap:require hyperscript.org` but I don't know how to get it "activated".

**Comment by back-2-95** [comment]
Maybe @javiereguiluz or @weaverryan knows?

**Comment by stof** [comment]
You need to import it in your page
